packages/wallet/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "../../config/tsconfig.json",
-  "include": ["src/**/*.ts", "test/**/*.ts"]
+  "include": ["src/**/*.ts", "test/**/*.ts"],
+  "compilerOptions": {
+    "moduleResolution": "Node16"
```

Comment:
Should we add this elsewhere?

Reply:
I'm going to do some cleanup on this branch.  some of these config edits might be unnecessary.

Reply:
Sounds good.